content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-kubeconfig.md

```diff
@@ -0,0 +1,24 @@
+---
+title: kubeadm kubeconfig
+content_type: conceito
+weight: 90
+---
+
+`kubeadm kubeconfig` fornece utilitários para gerenciar arquivos kubeconfig.
+
+Exemplos de como usar `kubeadm kubeconfig user`, consulte [Gerando arquivos kubeconfig para usuários adicionais](/docs/tasks/administer-cluster/kubeadm/kubeadm-certs#kubeconfig-additional-users).
```

Comment:
Maybe add a "para" at the beggining of this sentence ("para exemplos de uso do comando `kubeadm kubeconfig user`...").

Reply:
Nice. Perfect suggestion!